pallets/corporate-actions/src/lib.rs

```diff
@@ -834,6 +853,89 @@ decl_error! {
 }
 
 impl<T: Config> Module<T> {
+    fn unsafe_initiate_corporate_action(
```

Comment:
```suggestion
    fn unverified_initiate_corporate_action(
```
(reserving `unsafe_` for the `unsafe` meaning in Rust...)

Reply:
I have created a task for this where we can collect other code cleanup tasks like this: MESH-1797
Since there is a lot of those in the code and possibly other cleanup that needs to be done.